trunk/SUAVE/Methods/Noise/Correlations/noise_propeller_sae.py

```diff
@@ -0,0 +1,205 @@
+            FL1[id] = 6.8447630205*np.log(HP[id])+50.7481250789
+        
+        # ********************************** 
+            #Atmospheric Absorption and Spherical Spreading
+        # ***** Figure 5 *********************
+
+        #Perceived Noise Level and dbA
+        FL3_1[id] = -10.6203648721*np.log(dist[id]) + 63.9095683153
+        
+        #Overall sound pressure level
+        FL3_2[id] = -9.0388679389*np.log(dist[id]) + 55.9177440082
+        
+        # ********************************** 
+            # Directivity Index
+        # ***** Figure 6 *********************
+
+        DI[id] = 0.000000003*(theta[id]**5) - 1.5330661136843e-6*(theta[id]**4) + 0.0002748446*(theta[id]**3) - 0.0224465579*(theta[id]**2) + 0.8841215644*theta[id] -15.0851264829
+    
+        # **********************************
+            # PNL adjustment for 2 bladed propellers
+        # ***** Figure 7 *********************
+
+        if n_blades == 2:
+
+            if Vtip_Mach[id] <= 0.5:
+                PNL_factor[id] = -0.0002194325*(diameter)**4 + 0.0119121744*diameter**3 - 0.1715062582*diameter**2 - 0.440364203*diameter + 6.2098100491
+            
+            elif Vtip_Mach[id] <= 0.6:   
+                PNL_factor[id] = -0.000244323*(diameter)**4 + 0.0152241341*diameter**3 - 0.2925615622*diameter**2 + 1.1285073819*diameter + 1.4783088763
+            
+            elif Vtip_Mach[id] <= 0.7:   
+                PNL_factor[id] = -9.19201576386925e-5*(diameter)**4 + 0.007125071*diameter**3 - 0.1605227186*diameter**2 +0.5589157992*diameter + 2.4062117898
+            
+            elif Vtip_Mach[id] <= 0.8:
+                PNL_factor[id] = -9.71380207504857e-6*(diameter)**4 + 0.0013799409*diameter**3 - 0.0240602419*diameter**2 -0.6106811004*diameter + 6.8595599903
+            
+            elif Vtip_Mach[id] <= 0.85:
+                PNL_factor[id] = 0.0005259053*(diameter)**3 - 0.0058445988*diameter**2 -0.6296392506*diameter + 7.3325512636
+            
+            elif Vtip_Mach[id] <= 0.9:
+                PNL_factor[id] = 0.0001869235*(diameter)**4 - 0.0108618115*diameter**3 + 0.2292260658*diameter**2 - 2.468037691*diameter + 13.2430172278
+        else:
+            print('ERROR: Method limited for 2 bladed propellers right now!!')
+            return (0,0,0)
+        
+        # ****************** CALCULATION OF NOISE LEVELS *********************        
+        OASPL = FL1[id]+FL2+FL3_2[id]+DI[id]+NC
+        PNL[id] = OASPL + PNL_factor[id]
+        PNL_dBA[id] = PNL[id] - 14
+        
+        # *********************** END OF LOOP  *******************************    
+        
+    # Effective Perceived Noise Level for takeoff and landing:
+    EPNdB_takeoff = np.max(PNL) - 4
+    EPNdB_landing = np.max(PNL) - 2
+    
+    # ********************************************************    
+        # Write output file
+    # ********************************************************   
+
+    if ioprint:
```

Comment:
how often is this used?

Reply:
No much at the moment...but over the next year a lot. this is stuff that Carlos and Tarik did at Embraer. We should keep it in since its validated